translation-web/src/main/java/eu/europeana/api/translation/web/utils/PreProcessorUtils.java

```diff
@@ -0,0 +1,39 @@
+package eu.europeana.api.translation.web.utils;
+
+import eu.europeana.api.translation.definitions.model.TranslationObj;
+
+import java.util.List;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+public class PreProcessorUtils {
```

Comment:
this class is not really needed

Reply:
It's to initialize the pattern once. We can have this class have common constants and other contents for pre processor classes

Reply:
the patterns is common, but it is better to have each Preprocessor instantiating own instance of the pattern as the beans are singletons anyway